### ISSUE TITLE
Gleaner and Nabu progress logs are not helpful

### ISSUE THREAD

**Issue by webb-ben** [internetofwater/scheduler#60]
Inside Dagster, Gleaner and Nabu are printing the progress they are making. However it appears to get lost in the Dagster UI as a hex string instead of what should be a progress bar:

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/645361df-d94d-4514-96ac-6f246008eb99">

We should make sure this gets serialized as UTF-8 if we are going to be printing the logs to the UI. Otherwise I would request we remove this feature because it makes it hard to track the progress of the run.

**Comment by C-Loftus** [comment]
I agree with this. And maybe further we should remove progress bar stuff in general. Checking if we are in a headless env is flakey and adds another dependency.

**Comment by webb-ben** [comment]
Yeah seems like it was built for CLI monitoring... and doesn't translate into the web UI

**Comment by webb-ben** [comment]
fwiw decoding the logs to UTF-8 instead of latin does not help

**Comment by C-Loftus** [comment]
We should align on which program is more important to prioritize since both programs have a fair bit of logging.

**Comment by webb-ben** [comment]
Nabu is likely the worse culprit at the moment. Gleaner prints semi-regular (based on timestep) logs. Nabu prints 4-5 logs per item/transaction.

**Comment by C-Loftus** [comment]
Think this is addressed now. Future log issues can probably be added a new issue
